FoodBoard/Migrations/FoodBoardContextModelSnapshot.cs

```diff
@@ -107,12 +104,15 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<string>("Street")
                         .HasMaxLength(100);
 
+                    b.Property<string>("StreetNumber")
+                        .HasMaxLength(20);
```

Comment:
Teil mir mal bitte alle Eigenschaften zu denen mit. Das ist halt der ein Punkt der Frontend und Backend betrifft da solltest du nicht einfach alleine etwas festlegen.

Reply:
Wenn du das Backend in VS öffnest. Im Ordner Models findest du alle Entitäten die genutzt werden + die Annotations zu den Properties

Reply:
Okay aber dann sind bestimmt werte falsch gesetzt 

Straße: 50
Stadt: 26
Nummer: 5